cmd/tetra/tracingpolicy/tracingpolicy.go

```diff
@@ -20,9 +21,40 @@ func New() *cobra.Command {
 		Short:   "Manage tracing policies",
 	}
 
+	var tpModMode string
+	tpModCmd := &cobra.Command{
+		Use:   "modify <yaml_file>",
+		Short: "modify a tracing policy yaml file",
```

Comment:
nit: for using facing strings (help, errors) it would be better to write `YAML` instead of `yaml`, but this is just an acronym nit 😛

Reply:
fixed, thanks.